linear/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program solves a linear inequality Ax + B > 0.")
+	var A, B float64
+	fmt.Println("Enter A and B:")
+	fmt.Scanln(&A, &B)
+	if A < 0 {
+		x := -B / A
+		fmt.Println("x <", x)
+	} else if A > 0 {
+		x := -B / A
+		fmt.Println("x >", x)
+	} else if A == 0 {
```

Comment:
nit: You don't need `if` here. You've eliminated cases where A < 0 and A > 0, so it can be only 0 here.